config/settings.yml

```diff
@@ -83,6 +83,7 @@ sign_in:
 terms_of_use:
   current_version: v1
   provisioner_cookie_domain: localhost
+  enabled_clients: vaweb, mhv, myvahealth
```

Comment:
@rileyanderson, I just want to double check that this is the intended format and you don't need it to be a list like this?
```yaml
  enabled_clients:
    - vaweb
    - mhv
    - myvahealth
   ```

Reply:
Yeah, we have to account for how these will get loaded from parameter store. We unfortunately don't have a way to load arrays from parameter_store/env variables so this is the next best thing

Reply:
ok, no problem 👍